common/src/main/java/org/opensearch/ml/common/connector/HttpConnector.java

```diff
@@ -321,16 +321,23 @@ public <T> T createPayload(String action, Map<String, String> parameters) {
         Optional<ConnectorAction> connectorAction = findAction(action);
         if (connectorAction.isPresent() && connectorAction.get().getRequestBody() != null) {
             String payload = connectorAction.get().getRequestBody();
-            payload = fillNullParameters(parameters, payload);
```

Comment:
why are we removing this?

Reply:
added back in https://github.com/opensearch-project/ml-commons/pull/2811/commits/83572b53fe1615ec993e8a88b7ed7b150f5a5a5d